File: py/calculator.py
print("Calculator v1.0")
print("Copyright (c) 2020 miniprime1\n")

print("Options")
print("1. Addition")
print("2. Subtraction")
print("3. Multiplication")
print("4. Division")
choice = input("Enter choice: ")

print()
if choice == "1":
    x = float(input("Enter 1st input: "))
    y = float(input("Enter 2nd input: "))
    print("")
    print("Result")
    print(x, "+", y, "=", x + y)

elif choice == "2":
    x = float(input("Enter 1st input: "))
    y = float(input("Enter 2nd input: "))
    print("")
    print("Result")
    print(x, "-", y, "=", x - y)

elif choice == "3":
    x = float(input("Enter 1st input: "))
    y = float(input("Enter 2nd input: "))
    print("")
    print("Result")
    print(x, "*", y, "=", x * y)

elif choice == "4":
    x = float(input("Enter 1st input: "))
    y = float(input("Enter 2nd input: "))
    print("")
    print("Result")
    print(x, "/", y, "=", x/y if y!=0 else "Nan")
    
else:
    print("Error: Invalid Choice")

File: java/Calculator.java
import java.util.*;

public class Calculator {
    public static void main(String[] args) {
        var input = new Scanner(System.in);
        double x, y;

        System.out.println("Java Calculator v1.0");
        System.out.println("Copyright (c) 2020 miniprime1\n");
        System.out.println("Options");
        System.out.println("1. Addition");
        System.out.println("2. Subtraction");
        System.out.println("3. Multiplication");
        System.out.println("4. Division");
        System.out.print("Enter choice: ");
        int choice = Integer.parseInt(input.nextLine());
        System.out.println(' ');

        if (choice == 1) {
            System.out.print("Enter 1st input: ");
            x = Double.parseDouble(input.nextLine());
            System.out.print("Enter 2nd input: ");
            y = Double.parseDouble(input.nextLine());
            System.out.println("\nResult: " + x + " + " + y + " = " + (x+y));
        } else if (choice == 2) {
            System.out.print("Enter 1st input: ");
            x = Double.parseDouble(input.nextLine());
            System.out.print("Enter 2nd input: ");
            y = Double.parseDouble(input.nextLine());
            System.out.println("\nResult: " + x + " + " + y + " = " + (x-y));
        } else if (choice == 3) {
            System.out.print("Enter 1st input: ");
            x = Double.parseDouble(input.nextLine());
            System.out.print("Enter 2nd input: ");
            y = Double.parseDouble(input.nextLine());
            System.out.println("\nResult: " + x + " + " + y + " = " + (x*y));
        } else if (choice == 1) {
            System.out.print("Enter 1st input: ");
            x = Double.parseDouble(input.nextLine());
            System.out.print("Enter 2nd input: ");
            y = Double.parseDouble(input.nextLine());
            System.out.println("\nResult: " + x + " + " + y + " = " + (x/y));
        } else {
            System.out.println("Error: invalid choice");
        }
        
        input.close();
    }
}
